test

```diff
@@ -282,6 +300,15 @@ function dep_pass {
 	fi
 }
 
+function build_test_pass {
+	out="bin"
+	if [ -n "${BINDIR}" ]; then out="${BINDIR}"; fi
+	# Static compilation is useful when etcd is run in a container
+	PKGS=$TEST
+	ETCD_PKGS_DELIM=$(echo $PKGS | sed 's/ /,/g')
+	CGO_ENABLED=0 go test -c -covermode=set -coverpkg=$ETCD_PKGS_DELIM -o ${out}/etcd_test 
```

Comment:
why CGO_ENABLED=0? use a minimal set of flags when possible; also call this `etcd_cov`? test is not descriptive